deployment/dockerfiles/dev/event-listener

```diff
@@ -2,32 +2,25 @@
 
 FROM node:9 as origin-js-build
```

Comment:
For Travis CI tests, we use node:8 - would probably makes sense to be consistent ?

Reply:
Yep good plan, I will update these.